vscode-extensions.rokoroku.vscode-theme-darcula: init at 1.2.3

colmmurphyxyz opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

- Add new Vscode extension 'Darcula Theme' from author rokoroku at version 1.2.3 (latest)
  - Defined in `vscode/extensions/default.nix` as this is a basic color theme with no additional configuration needed.
- Create new entry 'colm' in maintainer-list.nix

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

colmmurphyxyz committed:
maintainers: add colm

qzylinra commented on pkgs/applications/editors/vscode/extensions/default.nix:
```diff
@@ -3927,6 +3927,22 @@ let
+          hash = "sha256-tlI+3DyDPo9tTOVTrEHoOtVTnEYSjgu9wojTXmEvv4E=";
+        };
+        meta = {
+          description = "A theme extension for VS Code based on Darcula theme from Jetbrains IDEs";
```
```suggestion
          description = "Theme extension for VS Code based on Darcula theme from Jetbrains IDEs";
```

colmmurphyxyz committed:
vscode-extensions.rokoroku.vscode-theme-darcula: init at 1.2.3

Release https://github.com/rokoroku/vscode-theme-darcula/commit/657baba5dc1d632daefe8e6130abcca550f0ea1f

colmmurphyxyz commented:
> please squash commits

I've squashed the two most recent commits on this branch into one (extension init and 'update phrasing...'), And kept the first commit that updated the maintainers list. Let me know if you want everything squashed together.

Thanks for the review!
